pkg/plugins/manager/client/client.go

```diff
@@ -204,3 +218,73 @@ func (s *Service) plugin(ctx context.Context, pluginID string) (*plugins.Plugin,
 
 	return p, true
 }
+
+// removeConnectionHeaders removes hop-by-hop headers listed in the "Connection" header of h.
+// See RFC 7230, section 6.1
+//
+// Based on https://cs.opensource.google/go/go/+/refs/tags/go1.19.4:src/net/http/httputil/reverseproxy.go;l=411-421
```

Comment:
I cannot open this link. It says permission denied. 
I think the link must be one of the followings:
- https://go.googlesource.com/go/+/refs/tags/go1.19.4/src/net/http/httputil/reverseproxy.go
- https://github.com/golang/go/blob/release-branch.go1.19/src/net/http/httputil/reverseproxy.go